inference-engine/include/ie_blob.h

```diff
@@ -577,13 +577,13 @@ class TBlob : public MemoryBlob {
     /**
      *@brief Virtual destructor.
      */
-#ifdef __clang__
-    virtual ~TBlob();
-#else
+//#ifdef __clang__
```

Comment:
We shouldn't have a commented code in OpenVINO headers.

BTW I am not sure that we won't have issues with RTTI on macOS.

Reply:
may be we need to use different approach: export not only for clang, but for gcc as well. In this case binaries compiled with gcc can be used with clang after

CC @dkurt @IRDonch

Reply:
I think my fix is a kind of temporary workaround and there might be more better way to solve the issue. I don't stick to my solution.
BTW, I attached the source code and cmake file which I compiled with DPC++ compiler for issue reproduction, just for in case.
[openvino-cnn.zip](https://github.com/openvinotoolkit/openvino/files/5936234/openvino-cnn.zip)

Reply:
This fix will break Android setup. I'd like to recommend to keep just a definition `virtual ~TBlob();` but remove conditional compilation for TBlob's destructor at https://github.com/openvinotoolkit/openvino/blob/e1a7f7e5af2916ee6831972a27b8cf6c585efa26/inference-engine/src/inference_engine/ie_rtti.cpp#L105-L109

Reply:
I tried to leave 'virtual ~TBlob();' but the 'unresolved reference' error occur again.
Compile successfully done with 'virtual ~TBlob() {};' on both DPC++ and MSVC2019 compilers, so I committed this change and pushed this as fix #2.

Reply:
@dkurt am I right that it does not solve the issue CVS-41513 which is exactly the same as this one, but for different compiler. Can we have generic solution?

Reply:
@ilya-lavrenov, I would like to remove conditional compilation at all.